samples/winapi_filefind/main.c

```diff
@@ -1,33 +1,20 @@
 #include <stdio.h>
 #include <string.h>
 #include <windows.h>
+#include <nxdk/mount.h>
 #include <hal/debug.h>
 #include <hal/video.h>
 #include <hal/xbox.h>
 
-int mount_drive_c ()
-{
-    OBJECT_STRING MountPath = RTL_CONSTANT_STRING("\\??\\C:");
-    OBJECT_STRING DrivePath = RTL_CONSTANT_STRING("\\Device\\Harddisk0\\Partition2\\");
-    NTSTATUS status;
-
-    status = IoCreateSymbolicLink(&MountPath, &DrivePath);
-    if (!NT_SUCCESS(status)) {
-        debugPrint("Failed to mount C: drive!\n");
-        Sleep(5000);
-        return -1;
-    }
-
-    return 0;
-}
-
 int main(void)
 {
     XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
     // Mount C:
-    int ret = mount_drive_c();
-    if (ret != 0) {
+    BOOL ret = XMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
+    if (!ret) {
+        debugPrint("Failed to mount C: drive!\n");
+        Sleep(5000);
```

Comment:
Note: We could avoid this by just adding more files to the XBE path and querying it.

Reply:
True, but I think it makes sense to include a simple mounting demonstration here (the downside is that the output of this depends on the user's system, so it's not perfect as a test).

Reply:
I think that mounting should be it's only sample in the future (not necessarily in this PR).
For the moment, I'd personally just prefer to see our samples simplified and reduced to "do one thing, but do it well".

However, we can also do that in a follow-up PR which contributes a new "mount"-sample.